app/screens/settings-screen/account-screen.tsx

```diff
@@ -81,11 +81,13 @@ export const AccountScreen = ({ navigation }: Props) => {
       console.error(err, "Failed to open WhatsApp")
 
       Linking.openURL(
-        `mailto:${CONTACT_EMAIL_ADDRESS}?subject=${LL.support.deleteAccount()}&body=${LL.support.deleteAccountFromPhone(
-          {
+        `mailto:${CONTACT_EMAIL_ADDRESS}?subject=${encodeURIComponent(
```

Comment:
`encodeURIComponent` this is what make the `mailto:` works?

Reply:
Yes it is. I can fix the other mailto instances.